src/js/components/receiving/PartialReceivingPage.jsx

```diff
@@ -261,15 +261,38 @@ const TABLE_FIELDS = {
       unitOfMeasure: {
         type: (params) => <LabelField {...params} />,
         label: 'react.partialReceiving.shippedInPo.label',
-        defaultMessage: 'Shipped (in PO)',
+        defaultMessage: 'Shipped (in PO UoM)',
         multilineHeader: true,
         flexWidth: '1',
         attributes: {
+          cellClassName: 'text-right',
           showValueTooltip: true,
         },
-        getDynamicAttr: (props) => ({
-          hide: !props.values?.isShipmentFromPurchaseOrder,
-        }),
+        getDynamicAttr: ({ values, parentIndex, rowIndex }) => {
+          const shipmentItem = _.get(
+            values,
+            `containers[${parentIndex}].shipmentItems[${rowIndex}]`,
+            {},
+          );
```

Comment:
can you use nullsafe here?

Reply:
I had some issues when using 
```
 const shipmentItem = values?.containers[parentIndex]?.shipmentItems[rowIndex] ?? {};
```

I guess the values are very deeply nested and nulish operator does not handle it very well